learn/getting_started/installation.md

```diff
@@ -95,6 +96,7 @@ cargo build --release
 :::
 
 ::: tab Windows
+If you are new to the Windows CMD, you can learn more about it [here](https://www.makeuseof.com/tag/a-beginners-guide-to-the-windows-command-line/).
 To install MeiliSearch on Windows, you can:
```

Comment:
I'm thinking something similar to my comment on lines 38-39 might be happening here: we first send people to a tutorial before we say anything regarding how it works in the context of MeiliSearch